unpackaged/config/lqa/settings/Security.settings

```diff
@@ -0,0 +1,517 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@davidmreed I am triple-checking that this is acceptable. Given that it is in LQA config and a necessary step of the process to open the orgs up given the way LQA is currently handled, this is a major boost in deployment and org creation speed for the team.

Reply:
How about if this task is only present in a local branch for the person who is preparing the LQA orgs?

I don't want this task to be exposed in a way that it could be run by users on other orgs, especially because EDA Is an open source project.

Reply:
@davidmreed that makes perfect sense. I thought if we put it in unpackaged we should be safe but apparently you made a good point of EDA as open source project.
@spinkelman @rhakeshj fyi